src/test/java/org/fcrepo/storage/ocfl/DefaultOcflObjectSessionTest.java

```diff
@@ -1637,6 +1638,7 @@ private void assertVersions(final List<OcflVersionInfo> actual, final String...
         final var versionNums = actual.stream()
                 .map(OcflVersionInfo::getVersionNumber)
                 .collect(Collectors.toList());
+        sort(versionNums);
```

Comment:
Instead of sorting here, you could change the next line to:

```java
assertThat(versionNums, containsInAnyOrder(expected));
```

Reply:
Thanks @pwinckles : I'll change it.